templates/base.tmpl

```diff
@@ -1,4 +1,5 @@
 #? stdtmpl | standard
+#import random_font_color
```

Comment:
I admire you trying to get your packages into this project, but I really don't think this should be accepted. White is perfectly fine.

Reply:
I am not trying to get my packages into this project, not trying to get my packages anywhere,
even more if another package that does the same and is better I would not even create it.

I will revert to all white over the weekend, but lots of coders hate all white background, 
specially me with 50" LCD hahaha :smile: I would use Stylish browser plugin then...
:cat: